compose-production.yml

```diff
@@ -21,18 +21,21 @@ webapp:
   links:
     - rabbitmq:rabbit
     - database
-    - email
     - celery
   environment:
     - APP_PORT=8000
     - APP_HOST=0.0.0.0
     - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@database/postgres
-    - DJANGO_EMAIL_USE_TLS=false
-    - DJANGO_EMAIL_HOST=email
-    - DJANGO_EMAIL_HOST_PASSWORD=osmaxx
-    - DJANGO_EMAIL_HOST_USERNAME=osmaxx
-    - DJANGO_EMAIL_PORT=25
+    - DJANGO_EMAIL_BACKEND=django.core.mail.backends.smtp.EmailBackend
+    - DJANGO_EMAIL_HOST=<changeme>
+    - DJANGO_EMAIL_HOST_PASSWORD='<changeme>'
+    - DJANGO_EMAIL_HOST_USER='<changeme>'
+    - DJANGO_EMAIL_PORT=<changeme>
+    - DJANGO_EMAIL_SUBJECT_PREFIX='<changeme>'
+    - DJANGO_EMAIL_USE_TLS=<changeme>
+    - DJANGO_EMAIL_USE_SSL=<changeme>
+    - DJANGO_EMAIL_TIMEOUT=<changeme>
```

Comment:
Would the project not run at all, or would it fail upon trying to send the first email? If the latter, I see no benefit in overriding here the (reasonably sane) defaults you've put into `osmaxx-py/config/settings/*.py`, only to force the operator to again overwrite them.